src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -11522,15 +11522,19 @@ private NullableFlowState GetValue(int index)
                 {
                     return NullableFlowState.NotNull;
                 }
+
                 Debug.Assert(index < Capacity);
                 index *= 2;
+                Debug.Assert((_state[index], _state[index + 1]) != (false, false));
+
                 var result = (_state[index], _state[index + 1]) switch
                 {
-                    (false, false) => throw ExceptionUtilities.Unreachable(),
+                    (false, false) => NullableFlowState.NotNull, // Should not be reachable
```

Comment:
Consider linking the bug.

Reply:
It's a good idea, but I'd rather not re-run CI for this comment. I'm investigating a root-cause fix today, so I won't forget to come back to this.